src/mscorlib/src/System/String.cs

```diff
@@ -1441,6 +1441,16 @@ internal String[] SplitInternal(char[] separator, int count, StringSplitOptions
 
             return s;
         }
+                
+        [System.Security.SecuritySafeCritical]  // auto-generated
+        unsafe internal int GetBytesFromEncoding(byte* pbNativeBuffer, int cbNativeBuffer,Encoding encoding)
```

Comment:
You can implement this as an extension method and pin the string - no need to make it a internal string method.